docs/src/security.md

```diff
@@ -293,28 +293,43 @@ CloudNativePG.
 : The instance manager requires to `update` and `patch` the status of any
   `Backup` resource in the namespace
 
-### Pod Security Policies
+### Security Contexts
 
-!!! Important
-    Starting from Kubernetes v1.21, the use of `PodSecurityPolicy` has been
-    deprecated, and as of Kubernetes v1.25, it has been completely removed. Despite
-    this deprecation, we acknowledge that the operator is currently undergoing
-    testing in older and unsupported versions of Kubernetes. Therefore, this
-    section is retained for those specific scenarios.
-
-A [Pod Security Policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/)
-is the Kubernetes way to define security rules and specifications that a pod needs to meet
-to run in a cluster.
-For InfoSec reasons, every Kubernetes platform should implement them.
-
-CloudNativePG does not require *privileged* mode for containers execution.
-The PostgreSQL containers run as `postgres` system user. No component whatsoever requires running as `root`.
-
-Likewise, Volumes access does not require *privileges* mode or `root` privileges either.
-Proper permissions must be properly assigned by the Kubernetes platform and/or administrators.
+A [Security Context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/)
+defines privilege and access control settings for a pod or container.
+
+CloudNativePG does not require *privileged* mode for container execution.
+The PostgreSQL containers run as the `postgres` system user. No component whatsoever requires running as `root`.
+
+Likewise, Volume access does not require *privileged* mode nor `root` privileges.
+Proper permissions must be assigned by the Kubernetes platform and/or administrators.
 The PostgreSQL containers run with a read-only root filesystem (i.e. no writable layer).
 
-The operator explicitly sets the required security contexts.
+The operator manages the setting of security contexts for all pods and containers of a PostgreSQL cluster.
+The [Seccomp Profile](https://kubernetes.io/docs/tutorials/security/seccomp/) to be used for the PostgreSQL
+containers can be configured with the `spec.seccompProfile` section of the `Cluster` resource. If this 
+section is left blank, the containers will use a seccompProfile `Type` of `RuntimeDefault`, that is, the
+container runtime default.
+
+The security context of PostgreSQL containers using the default `seccompProfile` will look like this:
+```
+securityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+    - ALL
+  privileged: false
+  readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  seccompProfile:
+    type: RuntimeDefault
+```
+
+#### RedHat OpenShift SCC
```

Comment:
CNP doesn't support RedHat OpenShift, mealning that it shouldn't be mentioned in the documentation